src/ProcessedCodeCoverageData.php

```diff
@@ -0,0 +1,137 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of phpunit/php-code-coverage.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace SebastianBergmann\CodeCoverage;
+
+use SebastianBergmann\CodeCoverage\Driver\Driver;
+
+/**
+ * Processed/context-added code coverage data for SUT.
+ */
+final class ProcessedCodeCoverageData
+{
+    /**
+     * Line coverage data.
+     * An array of filenames, each having an array of linenumbers, each executable line having an array of testcase ids.
+     *
+     * @var array
+     */
+    private $lineCoverage = [];
+
+    public static function __set_state(array $serialisedData)
```

Comment:
Why do you think we need a `__set_state()` method? A value object must not know about persistence or serialization.

Reply:
It's because of https://github.com/sebastianbergmann/php-code-coverage/blob/master/src/Report/PHP.php#L37

As this is now an object and not an array, the output from `var_export` automatically includes a call to `__set_state` which means it needs to exist as a method for the generated code to continue working.

I could change the PHP export to use `serialize`/`unserialize` instead of `var_export` instead?

Reply:
Do not worry about the PHP report. We can always bump the major version number and change the output of the PHP report.

Reply:
`__set_state` has been removed